storey/table.py

```diff
@@ -15,55 +21,50 @@ class Table:
     def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool = True):
         self._container, self._table_path = _split_path(table_path)
         self._storage = storage
-        self._cache = {}
-        self._aggregation_store = None
+        self._attrs_cache = {}
         self._partitioned_by_key = partitioned_by_key
+        self._aggregates = None
+        self._schema = None
+        self._read_only = False
```

Comment:
Isn't `read_only` only affect the aggregations? either rename it or move it somewhere else